app/javascript/components/mappers/domainToApiFields.js

```diff
@@ -1,81 +0,0 @@
-import { shapeMapper, listMapper } from '~/utils/objectUtils'
```

Comment:
@fddayan I have deleted this file per your mention that it is obsolete. I confirmed that it is not being used anywhere in the code before I deleted it.